### PR TITLE
Support observations with progTime and partTime

### PR DESCRIPTION
**Pull request by jdnavarro**
Now it's possible to import a proposal from the latest PIT which includes the `progTime`s and `partTime`s XML tags and then export it from ITAC.

This PR also includes the SQL schema migration script.

### CODE REVIEW

**Comment by cquiroz** [comment]
Congratulations on your first ITAC PR 👍

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java [hunk 1]
```diff
@@ -599,6 +599,8 @@ public ProposalBuilder addObservation(Band b, BlueprintBase bp, Condition c, Tar
         // put together the observation
         Observation o = new Observation();
         o.setBand(b);
+        o.setProgTime(new TimeAmount(0, TimeUnit.HR));
```
Comment (cquiroz) [inline phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: Why 0? This is probably not very important as the test is broken anyway

File: phase1-services/src/main/java/edu/gemini/tac/service/ProposalHibernateService.java [hunk 2]
```diff
@@ -743,12 +743,6 @@ private boolean editObservations(final String naturalId,
             Validate.notNull(band);
             observation.setBand(band);
         }
-        if (field.equals("observationTime")) {
```
Comment (cquiroz) [inline phase1-services/src/main/java/edu/gemini/tac/service/ProposalHibernateService.java]: We'll have to remove this as part of another PR. I guess it is ok to make the change here anyway

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java (hunk 1 above)
Comment (jdnavarro) [inline phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: If we set the times from parameters passed to `addObservation`, we'd break the function wherever it's called, and we might have to pass 0 anyway.

We could use the time parameter that is used for totalTime for both progTime and partTime, but I don't know if that makes sense. In the sample XML files I got I saw several instances where `progTime` and `partTime` were 0.

File: phase1-services/src/main/java/edu/gemini/tac/service/ProposalHibernateService.java (hunk 2 above)
Comment (jdnavarro) [inline phase1-services/src/main/java/edu/gemini/tac/service/ProposalHibernateService.java]: I could leave it commented out if you prefer.

Comment (cquiroz) [inline phase1-services/src/main/java/edu/gemini/tac/service/ProposalHibernateService.java]: 👍

File: phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java (hunk 1 above)
Comment (cquiroz) [inline phase1-data/src/test/java/edu/gemini/tac/persistence/fixtures/builders/ProposalBuilder.java]: Ok, that's fine.

**Comment by jdnavarro** [comment]
> Did you try displaying the proposals' pages?

I checked the HTML pages but I couldn't find any glaring error. However I'm not sure what to look for. Do the `progTime` and `partTime` have to be somehow rendered on those pages?

Also, I don't know if the sample XML file I'm using is realistic enough to be certain it will work properly in production.

> Maybe we should run a queue to make sure nothing else is broken

Not sure how to do that...

**Comment by cquiroz** [comment]
Don't worry about running queues, we can do that later together

**Pull request merged**